### ISSUE TITLE
Why does microsoft ignore all the winui performance complaints?

### ISSUE THREAD

**Issue by PunchCakee** [microsoft/microsoft-ui-xaml#8611]
### Describe the bug

This is very unprofessional behavior when you make a UI framework and just straight up ignore all the complaints about performance which is something very critical when you base windows 11 apps off of it!

https://github.com/microsoft/microsoft-ui-xaml/issues/4713
https://github.com/microsoft/microsoft-ui-xaml/issues/2032

this benchmark backs up my claims as well
https://github.com/Noemata/XamlBenchmark

even users are experiencing complaints with proof:
https://www.reddit.com/r/windowsinsiders/comments/116yxtn/whys_task_manager_so_hecin_slow/

### Steps to reproduce the bug

Use any windows 11 app

### NuGet package version

None

**Comment by Aurumaker72** [comment]
> even users are experiencing complaints with proof:
https://www.reddit.com/r/windowsinsiders/comments/116yxtn/whys_task_manager_so_hecin_slow/

The Windows 11 Task Manager is UWP + WinUI2 hosted in a Win32 application via a XAML Island.
Those performance issues aren't related to WinUI3.

**Comment by PunchCakee** [comment]
> > even users are experiencing complaints with proof:
> > https://www.reddit.com/r/windowsinsiders/comments/116yxtn/whys_task_manager_so_hecin_slow/
> 
> The Windows 11 Task Manager is UWP + WinUI2 hosted in a Win32 application via a XAML Island. Those performance issues aren't related to WinUI3.

so the performance issues are for winui 2?

**Comment by Aurumaker72** [comment]
No, WinUI2 is a visual styling library and doesn't have any inherent performance issues.

I can't comment on the internals and causes of the poor performance, but it's safe to assume that UWP is not the bottleneck here.
